packages/test-utils/src/set-watchers-to-sync.js

```diff
@@ -0,0 +1,30 @@
+function setDepsSync (dep) {
+  dep.subs.forEach((watcher) => {
+    watcher.sync = true
+    if (watcher.sync === true) {
```

Comment:
Is this right?  How does it work?

Reply:
You're right, this should be moved. We need a return statement so the code doesn't get stuck on circular deps, but this isn't the best place for it!